src/project.py

```diff
@@ -17,16 +17,30 @@ def lst():
 @echo_output
 def data(pname, output_format=None):
     "given a project name, returns the fully realized project description data."
-    ensure(output_format in [None, 'json', 'yaml'], "unknown output format %r" % output_format)
     formatters = {
         'json': core_utils.json_dumps,
         'yaml': core_utils.yaml_dumps,
         # None: core_utils.remove_ordereddict
         None: lambda v: v
     }
+    ensure(output_format in formatters.keys(), "unknown output format %r" % output_format)
     formatter = formatters.get(output_format)
     return formatter(project.project_data(pname))
 
+@task
```

Comment:
very useful to introspect the shape of `context`

Reply:
should probably support an environment too, e.g. `lax,end2end` to check for stuff that is related only to particular environments like RDS